common/src/main/scala/org/neo4j/spark/util/Validations.scala

```diff
@@ -1,17 +1,27 @@
 package org.neo4j.spark.util
 
 import org.apache.spark.internal.Logging
-import org.apache.spark.sql.SaveMode
+import org.apache.spark.sql.{SaveMode, SparkSession}
 import org.apache.spark.sql.types.StructType
 import org.neo4j.driver.{AccessMode, Session}
 import org.neo4j.spark.service.{Neo4jQueryStrategy, SchemaService}
 import org.neo4j.spark.util.Neo4jImplicits.StructTypeImplicit
-import org.neo4j.spark._
-
-import java.util.concurrent.Callable
 
 object Validations extends Logging {
 
+  def version(supportedVersions: Seq[String]): Unit = {
```

Comment:
Using varargs make the api more clearer:
```suggestion
  def version(supportedVersions: String*): Unit = {
```